TagTool/Commands/Porting/PortTagCommand.HUD.cs

```diff
@@ -93,6 +93,14 @@ private ChudDefinition ConvertChudDefinition(ChudDefinition chudDefinition)
 
                 for (int bitmapWidgetIndex = 0; bitmapWidgetIndex < chudDefinition.HudWidgets[hudWidgetIndex].BitmapWidgets.Count; bitmapWidgetIndex++)
                 {
+                    if (chudDefinition.HudWidgets[hudWidgetIndex].BitmapWidgets[bitmapWidgetIndex].Name.Value == 17796)

```

Comment:
I don't like that I have to look up what string you're checking for here.

Reply:
I'll cut it out, it's nothing significant. If I remember correctly it's just the budget string in forge or something like that.

Reply:
Something along the lines of `Cachec
```suggestion
(chudDefinition.HudWidgets[hudWidgetIndex].BitmapWidgets[bitmapWidgetIndex].Name == CacheContext.GetStringId("sandbox_budget");
```

Reply:
That looks good, I'll update tomorrow, afk for the night.

Reply:
Updated.